src/settings.js

```diff
@@ -29,6 +29,18 @@ export default {
     defaultColor: "#CCCCCC",
     initialCameraPosition: [-50, 0, 10],
 
+    areLabelsInitiallyShown: false,
+    labels: {
```

Comment:
Maybe `labelsConfig` since it's a key-value object?

Reply:
Agreed.